src/app/containers/IndexPageSection/utilities/filterAllowedItems.jsx

```diff
@@ -8,16 +8,13 @@ const MAX_ALLOWED_ITEMS_FIRST_SECTION = 13;
 const MAX_ALLOWED_ITEMS = 10;
 
 export const getAllowedItems = ({ items, isFirstSection, showAllPromos }) => {
-  let allowedItems = [];
-
+  if (showAllPromos) {
+    return items;
+  }
   if (isFirstSection) {
-    allowedItems = items.slice(0, MAX_ALLOWED_ITEMS_FIRST_SECTION);
-  } else if (showAllPromos) {
-    allowedItems = items;
-  } else {
-    allowedItems = items.slice(0, MAX_ALLOWED_ITEMS);
+    return items.slice(0, MAX_ALLOWED_ITEMS_FIRST_SECTION);
```

Comment:
Just to be clear, I can't see anywhere what the maximum allowed number is meant to be, just want to check it against our requirements? @jimjohnsonrollings can you confirm 13 is what we want?

Reply:
On FIXes, we want no limits.  On Front Pages and other IDXes, the limit is 13 for Top Stories.

Reply:
This looks like FIXes also have the limit of 13

Reply:
Just had another look at this, so `showAllPromos` takes precedence as this will always be set to `true` on the first sections is that right? 

Does that though mean that this code is never actually reached in production? Or is this component used for IDXes and FIXes?

Reply:
The default is 13, however with 'showAllPromos' this over-rides the limit for FIX pages. An examle is here: http://localhost:7080/mundo/noticias-internacional-53826365.